src/workers/parser/frameworks.js

```diff
@@ -22,12 +20,21 @@ function importsReact(imports) {
   );
 }
 
+function requiresReact(callExpressions) {
+  return callExpressions.some(
+    callExpression =>
+      callExpression.name === "require" &&
+      callExpression.arguments.some(arg => arg.value === "react")
+  );
+}
```

Comment:
nice!

for the record, this is another way it was done. not sure which is better

```diff
+ function importsReact(imports, identifiers) {
+   return (
+     imports.some(
+       importObj =>
+         importObj.source === "react" &&
+         importObj.specifiers.some(specifier => specifier === "React")
+     ) || identifiers.some(identifier => identifier.name === "React")

```